test/general/sycl_iterator.pass.cpp

```diff
@@ -3458,15 +3458,14 @@ int b[] = {0, 1, 1, 6, 6, 9};
 int c[] = {0, 1, 6, 6, 6, 9, 9};
 int d[] = {7, 7, 7, 8};
 int e[] = {11, 11, 12, 16, 19};
-constexpr auto na = sizeof(a) / sizeof(a[0]);
-constexpr auto nb = sizeof(b) / sizeof(b[0]);
-constexpr auto nc = sizeof(c) / sizeof(c[0]);
-constexpr auto nd = sizeof(d) / sizeof(d[0]);
+constexpr size_t na = sizeof(a) / sizeof(a[0]);
+constexpr size_t nb = sizeof(b) / sizeof(b[0]);
+constexpr size_t nc = sizeof(c) / sizeof(c[0]);
+constexpr size_t nd = sizeof(d) / sizeof(d[0]);
 
-template <typename Size>
-Size get_size(Size n)
+constexpr size_t get_size()
```

Comment:
m.b. better to write 
`constexpr size_t n = na + nb + nc + nd;` ?

Reply:
Fixed